common/table.cpp

```diff
@@ -83,6 +83,48 @@ void Table::del(std::string key, std::string /* op */)
                            "DEL operation failed");
 }
 
+bool Table::getField(std::string key, std::string field, std::string &value)
+{
+    redisReply *reply = (redisReply*)redisCommand(
+            m_db->getContext(),
+            "HGET %s %s",
+            getKeyName(key).c_str(),
+            field.c_str());
+
+    if (reply->type != REDIS_REPLY_STRING)
+    {
+        freeReplyObject(reply);
+        return false;
+    }
+
+    value = std::string(reply->str);
+
+    freeReplyObject(reply);
+
+    return true;
+}
+
+void Table::setField(std::string key, std::string field, std::string value)
+{
+    FieldValueTuple entry(field, value);
+
+    std::vector<FieldValueTuple> values { entry };
+
+    set(key, values);
+}
+
+void Table::delField(std::string key, std::string field)
+{
+    RedisReply r(m_db, string("HDEL ") + getKeyName(key) + " " + field, REDIS_REPLY_INTEGER);
```

Comment:
Please use redisFormatCommand. (See formatHSet API for exmaple)
'field' can be anything...